src/cim/components/ClusterDeployment/ClusterDeploymentWizardFooter.tsx

```diff
@@ -96,30 +79,18 @@ const ClusterDeploymentWizardFooter = ({
       }, []),
     [agents],
   );
-  const isClusterReady = React.useMemo(
-    () =>
-      clusterStatus &&
-      getWizardStepClusterStatus(
-        currentStepId,
-        wizardStepsValidationsMap,
-        { status: clusterStatus, validationsInfo: validationsInfo || {} },
-        hosts,
-      ) === 'ready',
-    [clusterStatus, currentStepId, hosts, validationsInfo],
-  );
 
   const alertsSection = alerts.length ? <Alerts /> : undefined;
-  const errorsSection =
-    !!errorFields.length || requireProxy || !isClusterReady ? (
-      <ValidationSection
-        errorFields={errorFields}
-        requireProxy={requireProxy}
-        currentStepId={currentStepId}
-        validationsInfo={validationsInfo}
-        clusterStatus={clusterStatus}
-        hosts={hosts}
-      />
-    ) : undefined;
+  const errorsSection = (
```

Comment:
This reintroduces the problem with the footer height since the AlertGroup is always rendered and takes space.

Reply:
Let's fix this in a follow up